### ISSUE TITLE
Pre-Train Model

### ISSUE THREAD

**Issue by sarahxinxi** [bebory/hair_modeling#1]
I am interested the work of HairNet.I have some questions to ask you.
1.Did you have finish the training work ?Will you provide the pre-trained model for inference.
2.Did you perform preprocessing on the trainning data, such as representing each strand in the local coordinate of anchored on its root and normalization. Could you give some examples of you training data, so that I can know the preprocessing
3. How about the result for real image. Can the details of hair  be predicted? Or need some post processing such as the reconstruction mentioned in the paper.
